hybridse/src/udf/udf.cc

```diff
@@ -159,6 +159,10 @@ int32_t weekofyear(Date *date) {
 
 float Cotf(float x) { return cosf(x) / sinf(x); }
 
+double degree_to_radius(double degree){
+    return degree/180.0L*3.141592653589793238463L;
```

Comment:
can u use the `M_PI` variable defined in `cmath`.? also u may change the `pi` in `Degress` as well

Reply:
No, it will cause the precision problem. I will define a variable M_PI by myself.

Reply:
It has already defined, and I'll use it!